scripts/test-cli.sh

```
@@ -14,6 +14,7 @@ echo '{"compilerOptions":{"paths":{"@/*":["./*"]}}}' > tsconfig.json
 # Test init
 $CLI init -y
 test -f beaket.ui.json
+! grep -q '"\$schema"' beaket.ui.json
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)